test/views/tab_test.dart

```diff
@@ -30,24 +28,25 @@ void main() {
 
       int counter = 0;
       for (Widget widget in listOfWidgets) {
+        //TODO: making all those things like Today Fri flexible
         if (widget is Text) {
           if (counter == 0) {
-            expect(widget.data, 'Timetable');
+            expect(widget.data, 'Today');
             counter++;
           } else if (counter == 1) {
-            expect(widget.data, "Courses");
+            expect(widget.data, "Fri");
```

Comment:
Isn't this tests expecting, that today is always Fri?

Reply:
oh sry thank you i will change evreything

Reply:
I think it should be possible to get the current day by: new DateTime.now().day
Maybe that helps.